.ci/install_kata.sh

```diff
@@ -19,7 +19,7 @@ TEST_CGROUPSV2="${TEST_CGROUPSV2:-false}"
 
 echo "Install Kata Containers Image"
 echo "rust image is default for Kata 2.0"
-"${cidir}/install_kata_image.sh" "${tag}"
+osbuilder_distro="$ID" "${cidir}/install_kata_image.sh" "${tag}"
```

Comment:
@fidencio hi, I am interested on having this merged.

I think this change should be made in the `"${cidir}/install_kata_image.sh"` script as 1) that is the script that should know about the "business rules" 2) someone (CI scripts for openshift does it) can call  `"${cidir}/install_kata_image.sh"` directly.

Also I think `osbuilder_distro` should default to `$ID` unless it was already set.

Reply:
> Let me re-work on this Tomorrow morning, @wainersm!
> 
> I finally have a working environment and I will start going through my backlog of things by this one.

I forgive you ;)

Reply:
I updated the branch and re-pushed it.
I'm going to trigger `/test` to see what will break and we can have a chat Tomorrow about this.